setup.py

```diff
@@ -15,16 +15,16 @@
 history = open("CHANGES.rst").read()
 
 tests_require = [
-    "check-manifest>=0.25",
-    "isort>=4.3",
-    "pydocstyle>=5.0.0",
-    "pytest>=4.6.1",
+    "pytest-cov>=2.10.1",
+    "check-manifest>=0.42",
+    "pytest-isort>=1.2.0",
+    "pytest-pycodestyle>=2.2.0",
+    "pytest-pydocstyle>=2.2.0",
+    "pytest>=6,<7",
```

Comment:
minor: do we need this upper bound?

Reply:
Since `pytest-invenio` is the central repo to define test dependencies, I just mirrored [what it currently has in its `setup.py`](https://github.com/inveniosoftware/pytest-invenio/blob/798eaa4140dcdfe6e2cb51338915701a993545ee/setup.py#L58) (Note that `pytest-invenio` depends on `docker-services-cli` that's why the duplication).